### PR TITLE
New Physics Selection Container with fix for EJE trigger in 2012

### PR DESCRIPTION
**Pull request by mfasDa**
In addition adapt the macro BrowseAndFillPhysicsSelectionOADB.C
- Add missing forward declarations
- Move string literals to const char *
- Add explicit cast to AliVEvent::EOfflineTriggerTypes when
  bitwise OR is used.

### CODE REVIEW

File: OADB/macros/BrowseAndFillPhysicsSelectionOADB.C [hunk 1]
```diff
@@ -622,12 +639,12 @@ AliOADBPhysicsSelection* Pbp2016(char* name = "oadb_Pbp_2016"){
...
 }
 
-AliOADBPhysicsSelection* pp2010(char* name = "oadbLHCpp2010"){
+AliOADBPhysicsSelection* pp2010(const char* name){
   // pp2010-11
   AliOADBPhysicsSelection * oadbLHCpp2010 = new AliOADBPhysicsSelection(name);
   UInt_t triggerCount = 0;
   oadbLHCpp2010->AddCollisionTriggerClass(AliVEvent::kINT1,"+CINT1B-ABCE-NOPF-ALL,CINT1-B-NOPF-ALL[NOTRD|]","B",triggerCount);
-  oadbLHCpp2010->AddCollisionTriggerClass(AliVEvent::kINT1 | AliVEvent::kFastOnly,"+CINT1-B-NOPF-FASTNOTRD -CINT1-B-NOPF-ALLNOTRD","B",triggerCount);
+  oadbLHCpp2010->AddCollisionTriggerClass((AliVEvent::EOfflineTriggerTypes)(AliVEvent::kINT1 | AliVEvent::kFastOnly),"+CINT1-B-NOPF-FASTNOTRD -CINT1-B-NOPF-ALLNOTRD","B",triggerCount);
```
Comment (mfasDa) [inline OADB/macros/BrowseAndFillPhysicsSelectionOADB.C:647]: Explicit cast is necessary in ROOT6 as rvalue (AliVEvent::kINT1 | AliVEvent::kFastOnly) is int. However in ROOT5 the cast should have been done explicitly, so the logics should not have changed.

**Comment by mfasDa** [comment]
@ekryshen @jgrosseo please review!

**Comment by mfasDa** [comment]
Could someone please have a look at the pull request (DPG AOT event selection)? At the moment the physics selection kills the EJE triggers in 2012 due to the trivial bug.

**Comment by fprino** [comment]
@iarsene, @ekryshen can you please have a look?
Thanks!

  Francesco

**Comment by mfasDa** [comment]
For the record: This relates to #14909.

**Comment by iarsene** [comment]
Hi @mfasDa and @fprino,
From my side the update is fine. I had to fix similar issues to comply with ROOT6.
I don't know, you may want to also wait for input from @ekryshen , but the changes should be harmless.
Cheers, Ionut

**Comment by mfasDa** [comment]
Hi @iarsene ,

Thanks for your reply! In principle I would have preferred that the original author have a quick look whether this is fine, not that something was overlooked (not sure there are tests implemented that check everything is fine). However there are people working on the jet spectra at 8 TeV with the EMCAL trigger which wait for their train run, so it would be good to have this fixed merged by today.

Cheers

Markus

**Comment by chiarazampolli** [comment]
+1

**Comment by chiarazampolli** [comment]
I +1-ed the PR, after Ionut's comment, and considering that it is urgent. But please, check ASAP, and possibly report here.

Chiara

**Pull request merged**